sdk/python/kfp/dsl/_types.py

```diff
@@ -19,72 +19,63 @@ class BaseType:
 
 # Primitive Types
 class Integer(BaseType):
-	openapi_schema_validator = {
+	openAPIV3Schema = {
```

Comment:
Could you give me some context on this change?

Reply:
I changed the key name to openAPIV3Schema for two reasons:
1) consistent with the static-config design of Pascal.
2) as a conventional name, openAPIV3Schema looks more official. 
WDYT?

Reply:
I feel it's better to follow python naming convention and only use the camel case name when serializing it to yaml or json.

Reply:
This comment is actually nit. I will let you decide.

/lgtm

Reply:
Maybe revert to the lower case with underscores, as indicated in the google python style guide(http://google.github.io/styleguide/pyguide.html)?

Reply:
And, having the yaml format the same lower case with underscore?

Reply:
Done.